services/user-management/src/main/java/com/foodie/user/model/User.java

```diff
@@ -28,7 +28,7 @@
 @AllArgsConstructor
 @Entity
 @Table(name = "users")
-public class User extends FoodieBaseModel implements UserDetails {
+public class User extends FoodieBaseEntity implements UserDetails {
```

Comment:
You're keeping the `implements userDetails`?

Reply:
Working on moving it. Still not certain the best way yet.